cmd/boilerplate.go

```diff
@@ -138,6 +166,30 @@ local config = require("` + pluginName + `.config")
 	}
 	defer mainLuaFile.Close()
 	mainLuaFile.WriteString(`local config = require("` + pluginName + `.config")`)
+
+	healthLua := filepath.Join(luaPath, "health.lua")
+	healthLuaFile, healthErr := os.Create(healthLua)
+	if healthErr != nil {
+		log.Fatal(healthErr)
+	}
+	defer healthLuaFile.Close()
+	healthLuaFile.WriteString(`local function checkhealth()
+   if vim.fn.has('nvim-0.5') then
+	  vim.fn['health#report_ok']('nightly installed')
+   else
+	  vim.fn['health#report_warn']('install neovim nightly')
+   end
+
+   -- check more health conditions here
+--[[    if vim.fn.executable('lua') ~= 1 then
```

Comment:
This is the right way to check, however 'lua' as example might be a bit confusing. (luaJIT is embedded in neovim, so you can call lua in nvim without having lua installed system-wide')

I'd rather see a less ambiguous example, maybe 'curl' or 'gcc' ? Not to say every plugin depends on that but this way one can change it with trust it won't break something.

Reply:
You're right! I replaced the example with `gcc` [here](https://github.com/gennaro-tedesco/boilit/blob/47bb0934e09cb354f2575ac796a0cb3d49eed24c/cmd/boilerplate.go#L128)